pkg/network/go/bininspect/symbol_filter.go

```diff
@@ -69,34 +69,36 @@ func (f stringSetSymbolFilter) findMissing(symbolByName map[string]safeelf.Symbo
 	return missingSymbols
 }
 
-// prefixSymbolFilter is a symbol filter which gets any one symbol which has the
-// specified prefix.
-type prefixSymbolFilter struct {
-	prefix    string
+// infixSymbolFilter is a symbol filter which gets any one symbol which has the
+// specified infix.
```

Comment:
> which gets any one symbol

I think the documentation should be rephrased

Reply:
Done.